crates/kilt-dip-primitives/Cargo.toml

```diff
@@ -11,13 +11,17 @@ repository    = { workspace = true }
 version       = { workspace = true }
 
 [dev-dependencies]
-cumulus-pallet-parachain-system = { workspace = true, features = ["std"] }
-enum-iterator                   = { workspace = true }
-hex-literal                     = { workspace = true }
-peregrine-runtime               = { workspace = true, features = ["std"] }
-rococo-runtime                  = { workspace = true, features = ["std"] }
-sp-io                           = { workspace = true, features = ["std"] }
-spiritnet-runtime               = { workspace = true, features = ["std"] }
+cumulus-pallet-parachain-system = { workspace = true, features = [
+  "parameterized-consensus-hook",
+  "std",
+] }
+cumulus-primitives-core = { workspace = true }
+enum-iterator = { workspace = true }
+hex-literal = { workspace = true }
+peregrine-runtime = { workspace = true, features = ["std"] }
+rococo-runtime = { workspace = true, features = ["std"] }
+sp-io = { workspace = true, features = ["std"] }
+spiritnet-runtime = { workspace = true, features = ["std"] }
```

Comment:
I guess we have to add some rules for taplo

Reply:
I could not find a smooth way to fix the issue, nor do I understand why it is happening.

For now, it is resolved by separating the cumulus-pallet-parachain-system crate from the other dependencies.